infra/ansible/config/env.yaml

```diff
@@ -44,6 +46,8 @@ build_env:
   cuda:
     TF_CUDA_COMPUTE_CAPABILITIES: 7.0,7.5,8.0
     XLA_CUDA: 1
+    PATH: /usr/local/cuda-{{ cuda_version }}/bin:/usr/local/nvidia/bin${PATH:+:${PATH}}
+    LD_LIBRARY_PATH: /usr/local/cuda-{{ cuda_version }}/lib64:/usr/local/cuda-{{ cuda_version }}/lib:/usr/local/nvidia/lib64:/usr/local/nvidia/lib${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}:/usr/local/lib
```

Comment:
If these variables are only needed during runtime (not build process), I'd removed these lines (build env variables). Other than that it looks good now.

Reply:
Thanks. They are not needed during build time. Instead, they're only needed during runtime. What's the proper place to add them?

Reply:
Btw, i stand corrected. It's needed for compiling PyTorch with cuda enabled.